### DISCUSSION TITLE
How to draw a stationary polygon on video panel

### DISCUSSION THREAD

**Discussion by Chenfengldw** [foxglove/community#1013] [Q&A]
Hi there, how can I draw a stationary polygon on the video panel? "Stationary" means this object stands still in the "real world" in the video. Thanks!

**Comment by banisadr** [comment]
You can use [`ImageAnnotation` messages](https://docs.foxglove.dev/docs/visualization/panels/image#imageannotation) to draw polygon's on the Image panel.

When you say "stationary" in the "real world" are you expecting the polygon to automatically track the object? Foxglove does not provide any object detection or tracking, so you will have to position the polygons yourself.

**Reply by achim-k** [reply]
If with "real world", you mean that the polygon should use transform information, then this is possible. You can select any renderable topic in the image panel settings and it will be visualized on top of the image.

Example:

[Screencast from 12.07.2024 12:54:07.webm](https://github.com/user-attachments/assets/afe81ccf-325b-4f6e-b28d-8fe841a3c0df)
